docs/maintainers/growing-your-project.md

```diff
@@ -18,33 +18,36 @@ When it comes to deciding on whether your open source project is ready to grow,
 - Are there tasks that are becoming too overwhelming for me and/or team?
 - How can I improve and maintaining my project's current and pending features?
 - How many stars and forks do I want to reach this time, and who can help me achieve it?
-If you have said yes to these questions, it's time to develop your growth strategies. If you haven't already, consider creating different teams for specific purposes. For example, you can create a docs team for documentation maintenance, a triage team for reviewing and improving issues, or a maintainer team to help you with overall oversight of the project.  Once you have decided the types of teams you want to add to your project, you can create separate pages for each team in your project's Workspace page. Check out the [Creating a New Repository Insights Page](../docs/maintainers-guide.md#adding-repositories-to-your-workspace).
+If you have said yes to these questions, it's time to develop your growth strategies. If you haven't already, consider creating different teams for specific purposes. For example, you can create a docs team for documentation maintenance, a triage team for reviewing and approving issues, or a maintainer team to help you with overall oversight of the project.  Once you have decided the types of teams you want to add to your project, you can create separate pages for each team in your project's Workspace page. Check out the [Creating a New Repository Insights Page](../docs/maintainers-guide.md#adding-repositories-to-your-workspace).
 
 ### How to identify new contributors and team members
 
-Like Superman, Wonder Woman, the Flash, and Batman in ***The Justice League***, the people you bring on your team(s) and your project's contributors are key to its success, which is why picking the right people is important. When doing this process, consider going to their OpenSauced profiles and ask yourself the following questions:
+Like Superman, Wonder Woman, the Flash, and Batman in ***The Justice League***, the people you bring on your team(s) and your project's contributors are key to its success, which is why picking the right people is important.  When doing this process, consider going to their OpenSauced profiles and checking their [dev card](../docs/features/dev-card.md). While you do this, ask yourself the following questions:
```

Comment:
We have to be careful here. Pop culture references are not accessible to everyone, and for our docs we want to make sure that a universal audience can understand and relate to them.

Reply:
I think, unlike blog posts, docs should be something "semi-formal" if not entirely "formal". Because docs is mainly talking about products/projects, and how to interact with them. Honestly, I've never seen docs written in an informal and non-universal way anywhere. My recommendation is to write this down in "semi-formal" style like what we have now.

Reply:
You'll notice in that post that I don't reference specific super heroes. I'm going to push back against the idea that batman and superman are universal. 

The concept of superheroes is always more universal than specifying them by name; however, there's a difference in audience and tone with docs. I consider our docs to be semi-formal and blog posts can be more conversational, because the audience likely varies from week to week.

Docs should be consistent and relevant at all times and for the broadest audience. So we need to be very deliberate here with the style and continue to maintain the style that we've established throughout the docs.